content/updates/productivity/3-A-more-helpful-process-tracker.md

```diff
@@ -0,0 +1,41 @@
+# More helpful process tracker
+
+## See all the ongoing jobs
+::: meta-data type="Improvement" ee-only="false" link-to-doc="../articles/monitor-jobs.html"
+:::
+
+To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the Process Tracker.
+
+You will now be able to see the ongoing or previous PIM jobs and, if needed, which users launched them. :wink:
+
+## Easily stop ongoing jobs
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/monitor-jobs.html#how-to-stop-your-jobs"
+:::
+
+A brand new `Stop` button is now displayed in your process tracker. It enables you to stop jobs: imports, exports but also ongoing mass actions or rule executions. Super helpful whenever you launch a job by mistake (we're only humans after all) and don't want to wait for it to run until the end. 😉
+
+![The new `Stop` button in the process tracker](../img/stop-your-jobs.png)
+
+This button comes with a dedicated permission `Stop an ongoing job`. If you need it, ask your PIM admin to give you the right to do so.
+
+## Monitor ongoing jobs progress and time left
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/monitor-jobs.html#what-can-you-find-in-a-job-detail"
+:::
+
+Whenever you launch a job, a progress bar and an estimation of the remaining time before the job is completed, are now displayed in the process tracker. You can now easily see how long it will take to process a job, to help you better organize your daily tasks.
+
+![Estimation of the time left and progress bar](../img/progress-bar-for-jobs.png)
+
+## Easily access to the detail of your quick export once launched
+::: meta-data type="Improvement" ee-only="false" link-to-doc="../articles/quick-export.html#how"
+:::
+
+When you launch a quick product export from the product grid, you can now easily jump to this task's progress detail in the process tracker. Indeed, you'll find a link to the job detail embedded in the flash message that confirms the quick export launch.
+
+## Process tracker display enhancements
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/monitor-jobs.html#searching-for-a-job"
+:::
+
+Quickly find a launched job thanks to the new `User` filter we introduce in the process tracker grid. It allows you to easily find either your own jobs or the ones launched by one of your colleagues. Neat!
+
+On a side note, you will also notice that you can sort the process tracker grid jobs by users.
```

Comment:
```suggestion
You are also now able to sort the process tracker grid jobs by users.
```

Reply:
Sounds really weird without any word to introduce this sentence. It's another improvement. We need something like "on a side note" or "in addition".

Reply:
```suggestion
Additionally, you are also able to sort the process tracker grid jobs by users.
```